Tests/TuistPluginTests/PluginServiceTests.swift

```diff
@@ -0,0 +1,134 @@
+import TSCBasic
+import TuistCore
+import TuistLoader
+import TuistLoaderTesting
+import TuistSupport
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistPlugin
+
+final class PluginServiceTests: TuistTestCase {
+    private var mockModelLoader: MockGeneratorModelLoader!
```

Comment:
I'd drop the `mock` prefix as it's redundant here IMHO -> it's a unit test, so it's safe to assume that all the implementations will be mock ones